trantor/net/EventLoop.cc

```diff
@@ -304,11 +303,11 @@ void EventLoop::wakeup()
     //     return;
     uint64_t tmp = 1;
 #ifdef __linux__
-    int ret = write(wakeupFd_, &tmp, sizeof(tmp));
+    __attribute__((unused)) int ret = write(wakeupFd_, &tmp, sizeof(tmp));
```

Comment:
__attribute__((unused)) is a GCC extension, please use `(void)ret` instead of it to avoid compilation errors if someone uses other compilers in Linux.

Reply:
@an-tao done!   the compilers that come to mind for linux -- gnu, clang and icc all accept `__attribute((unused))__`, but the void cast is more universal.  there's also `[[maybe_unused]]` for c++17, but that involves even more checks, so void it is.